src/btree/bt_sync.c

```diff
@@ -180,8 +217,11 @@ __sync_ref_is_obsolete(WT_SESSION_IMPL *session, WT_REF *ref)
  *     Evict the in memory page or delete the on disk page.
  */
 static int
-__sync_ref_evict_or_mark_deleted(WT_SESSION_IMPL *session, WT_REF *ref)
+__sync_ref_evict_or_mark_deleted(
```

Comment:
There's a line around 175 that has a bunch of unnecessary parenthesis:
```
mod = (ref->page != NULL) ? (ref->page->modify) : NULL;
```
I'd suggest:
```
mod = ref->page == NULL ? NULL : ref->page->modify;
```

Reply:
Fixed.